HUnit.cabal

```diff
@@ -30,7 +30,7 @@ library
   hs-source-dirs:
     src
   build-depends:
-    base ==4.*,
+    base >=4.9,
```

Comment:
Is Hackage accepting  packages that don't specify an upper bound on `base` by now?  I may be wrong on this, but I think that was not the case in the past?